.github/workflows/scanner-release-vuln-update.yaml

```diff
@@ -69,6 +72,7 @@ jobs:
         # Do not use the make target, as there may be some incompatibilities.
         # See https://github.com/stackrox/stackrox/pull/9227.
         cd scanner
+        go mod tidy
```

Comment:
I am not sure about this. 
Why are updates to `go.mod` necessary? (e.g. because of different go version locally vs GH runner? see https://github.com/actions/runner-images/issues/9326 -> a resolution for that would be to use the apollo-ci image instead).
Or any diff should be resolved and committed into the repository.